R/pkg/R/mllib.R

```diff
@@ -32,6 +32,9 @@ setClass("PipelineModel", representation(model = "jobj"))
 #' @param family Error distribution. "gaussian" -> linear regression, "binomial" -> logistic reg.
 #' @param lambda Regularization parameter
 #' @param alpha Elastic-net mixing parameter (see glmnet's documentation for details)
+#' @param standardize Whether to standardize features before training
+#' @param solver The solver algorithm used for optimization. Currently support "auto", "normal"
```

Comment:
Please explain `auto`, `normal`, and `l-bfgs`. If we are missing this info in the Scala API, we should add to both.